src/modules/commander/Commander.hpp

```diff
@@ -217,75 +191,104 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 
-	hrt_abstime	_datalink_last_heartbeat_gcs{0};
-	hrt_abstime	_datalink_last_heartbeat_avoidance_system{0};
-	hrt_abstime	_datalink_last_heartbeat_onboard_controller{0};
-	hrt_abstime	_datalink_last_heartbeat_parachute_system{0};
-	bool		_onboard_controller_lost{false};
-	bool		_avoidance_system_lost{false};
-	bool		_parachute_system_lost{true};
+	hrt_abstime _boot_timestamp{0};
+	hrt_abstime _last_disarmed_timestamp{0};
+	hrt_abstime _overload_start{0};		///< time when CPU overload started
 
-	hrt_abstime	_high_latency_datalink_heartbeat{0};
-	hrt_abstime	_high_latency_datalink_lost{0};
+	hrt_abstime _led_armed_state_toggle{0};
+	hrt_abstime _led_overload_toggle{0};
+
+	hrt_abstime _last_health_and_arming_check{0};
+
+	hrt_abstime _last_termination_message_sent{0};
+	hrt_abstime _last_valid_manual_control_setpoint{0};
+	hrt_abstime _last_wind_warning{0};
 
 	uint8_t		_battery_warning{battery_status_s::BATTERY_WARNING_NONE};
-	Hysteresis	_auto_disarm_landed{false};
-	Hysteresis	_auto_disarm_killed{false};
 
-	hrt_abstime	_last_print_mode_reject_time{0};	///< To remember when last notification was sent
-	bool            _mode_switch_mapped{false};
+	bool _failsafe_user_override_request{false}; ///< override request due to stick movements
 
-	bool		_last_overload{false};
+	// geofence
+	bool _geofence_loiter_on{false};
+	bool _geofence_rtl_on{false};
+	bool _geofence_land_on{false};
+	bool _primary_geofence_warning_action_on{false};
+	bool _primary_geofence_breached_prev{false};
 
-	bool		_is_throttle_above_center{false};
-	bool		_is_throttle_low{false};
+	bool _circuit_breaker_flight_termination_disabled{false};
 
-	hrt_abstime	_boot_timestamp{0};
-	hrt_abstime	_last_disarmed_timestamp{0};
-	hrt_abstime	_overload_start{0};		///< time when CPU overload started
+	bool _rtl_time_actions_done{false};
 
-	hrt_abstime _led_armed_state_toggle{0};
-	hrt_abstime _led_overload_toggle{0};
+	bool _flight_termination_triggered{false};
+	bool _imbalanced_propeller_check_triggered{false};
+	bool _lockdown_triggered{false};
 
-	hrt_abstime _last_termination_message_sent{0};
+	bool _avoidance_system_lost{false};
+	bool _onboard_controller_lost{false};
+	bool _parachute_system_lost{true};
+
+	bool _last_overload{false};
+	bool _mode_switch_mapped{false};
+
+	bool _is_throttle_above_center{false};
+	bool _is_throttle_low{false};
 
-	bool		_status_changed{true};
-	bool		_arm_tune_played{false};
-	bool		_was_armed{false};
-	bool		_have_taken_off_since_arming{false};
+	bool _arm_tune_played{false};
+	bool _was_armed{false};
+	bool _have_taken_off_since_arming{false};
+	bool _status_changed{true};
+
+	// commander_state_s	_commander_state{};
+	// geofence_result_s	_geofence_result{};
 
 	vehicle_land_detected_s	_vehicle_land_detected{};
-	vtol_vehicle_status_s	_vtol_vehicle_status{};
 
 	// commander publications
 	actuator_armed_s        _actuator_armed{};
 	vehicle_control_mode_s  _vehicle_control_mode{};
 	vehicle_status_s        _vehicle_status{};
```

Comment:
Note that I changed the order here as `_vehicle_status` is used above in `HealthAndArmingChecks _health_and_arming_checks{this, _vehicle_status};`, and to make sure `_vehicle_status` is initialzed before `HealthAndArmingChecks` is constructed. This could otherwise lead to uninitialized data access, so ordering is important.